core/src/main/java/com/amplifyframework/auth/CognitoCredentialsProvider.kt

```diff
@@ -76,10 +76,28 @@ internal class CognitoCredentialsProvider : AuthCredentialsProvider {
             )
         }
     }
+
+    fun getAccessToken(onResult: Consumer<String>, onFailure: Consumer<Exception>) {
```

Comment:
Which category needs `getAccessToken`? Consider adding this function to `AuthCredentialsProvider` interface too or remove it if not used.

Reply:
API category needs it. I will add it to the interface.